src/asim/configs/common/constants.yaml

```diff
@@ -2,40 +2,28 @@
 ## See full license in LICENSE.txt.
 
 scenarioYear: 2022
-# set below to 1 for everything to be internal, 0 otherwise
```

Comment:
All comments were removed from common/constants.yaml, should not be removing comments

Reply:
I suspect this may have happened when I tested running the scenario manager. Might be something we want to keep in mind.